elements/lisk-transactions/test/errors.ts

```diff
@@ -52,6 +52,61 @@ describe('errors', () => {
 					.to.have.property('dataPath')
 					.and.be.a('string');
 			});
+
+			it('should show provided actual property when present', () => {
+				TxError = new TransactionError(
+					'error message',
+					'transaction id',
+					'.dataPath',
+					'actual property',
+				);
+				return expect(TxError.toString()).to.match(/actual: actual property/);
+			});
+
+			it('should show provided expected property when present', () => {
+				TxError = new TransactionError(
+					'error message',
+					'transaction id',
+					'.dataPath',
+					'actual property',
+					'expected property',
```

Comment:
Just a general suggestion: I'd recommend using more random property names when testing these kinds of error handlings because somebody can write "expected property" in the error message. Eg. instead of `expected property` use `__EXPECTED_PROPERTY_1__`.